### ISSUE TITLE
The problem of prediction accuracy 

### ISSUE THREAD

**Issue by wojiaozhuyt** [matterport/Mask_RCNN#1161]
I trained my own dataset with 
                 steps=500,epochs=20,layers='heads'  and epochs=30, layers='all'.
When I validate my dataset, the results are very poor and the mAPs  are always 0.
What should I improve it?
Thanks for the replies.

**Comment by javierfs** [comment]
I have also faced that, and I solved it by following the following solutions: 

- To check that the configuration of the training and inference model are the same, except for the inference model in which the following variables should be : _GPU_COUNT = 1, IMAGES_PER_GPU = 1_

- To training enough time. If my learning rate is too small (10^-5), then after the 40 epoch is not starting to get some values)


hope it helps

**Comment by wojiaozhuyt** [comment]
@javierfs Thanks for your reply. I will try it right now.
